src/pg_to_evalscript/javascript_processes/linear_scale_range.js

```diff
@@ -0,0 +1,24 @@
+function linear_scale_range(arguments) {
+  const { x, inputMin, inputMax, outputMin = 0, outputMax = 1 } = arguments;
+
+  if (x === null || x === undefined) {
```

Comment:
You are right and it was changed.  I'll change other processes in separate PR.  
Btw, any idea how to share js code between processes? Only think I can think of is putting everything in some `common.js` file and load it together with each process.

Reply:
> Btw, any idea how to share js code between processes? Only think I can think of is putting everything in some common.js file and load it together with each process.

I'm not sure what you mean. Why would you share js code between processes? Can you give me an example?

Reply:
input validation, error codes and messages...

Reply:
Ah I see, makes sense. Haven't thought about that.

I can't think of a better idea either. So to recap:
1. We put a `common.js` file somewhere (`javascript_common`) directory or smth
2. We change `load_process_code` so that it also loads `common.js` and prepends it
3. We add it so that `common.js` is is prepended to the generated evalscript